package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
-    "fmt": "prettier --check '**/*.{json,md,yaml,yml}'",
-    "fmt:fix": "prettier --write '**/*.{json,md,yaml,yml}'",
+    "format:check": "prettier --check '**/*.{css,html,js,json,md,scss,ts,tsx}'",
```

Comment:
lmao, I just noticed that these formatters are different from the ones below in lint-staged :man_facepalming: I don't suppose there's some way to invoke one from the other or vice versa?

Reply:
Yea, having to remember to update both spots is not fun. I can't think of a way off the top of my head unfortunately :(

Reply:
If you find a way...that would be great. Maybe a raindy friday search topic!